src/lt/util/dom.cljs

```diff
@@ -37,92 +38,228 @@
     ([this n not-found]
      (or (.item this n) not-found))))
 
-(defn text-node [text]
+(defn text-node
+  "Create a text node containing string `text`."
+  [text]
   (js/document.createTextNode text))
 
 (defn $$
+  "Returns a NodeList of all elements within `elem` that matches `query`.
```

Comment:
To clarify, did you mean "+that *match"?

"within `elem` the match query" does not seem correct.